### DISCUSSION TITLE
starter 主题不支持评论功能

### DISCUSSION THREAD

**Discussion by xiaohaoxing** [YunYouJun/valaxy#460] [Q&A] [answered]
切换主题到 yun 或者 hairy 有评论区，用 starter 就没有。尝试看了一下源码，好像是暂时没有实现的？

**Comment by YunYouJun** [comment, marked as answer]
Starter 只是一个开发者模板，并没有实现主题的所有功能。
可以基于此自定义开发自己的主题。

**Reply by xiaohaoxing** [reply]
Got it! 
I'll try to develop the comment feature by myself based on the starter theme repo.
